src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Attributes.cs

```diff
@@ -298,29 +298,41 @@ public static void GetCustomAttributesWorksWithOpenAndClosedGenericTypesForEvent
 
         [Fact]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/56887", TestRuntimes.Mono)]
-        public static void GetCustomAttributesOnOpenGenericTypeRetrievesDerivedAttributes()
+        public static void GetCustomAttributesOnOpenGenericBaseTypeRetrievesDerivedAttributes()
         {
             Attribute[] attributes = Attribute.GetCustomAttributes(typeof(HasGenericAttribute), typeof(GenericAttribute<>));
-            Assert.Equal(3, attributes.Length);
-            Assert.Equal(1, attributes.Count(a => a.GetType() == typeof(DerivesFromGenericAttribute)));
-            Assert.Equal(1, attributes.Count(a => a.GetType() == typeof(GenericAttribute<bool>)));
-            Assert.Equal(1, attributes.Count(a => a.GetType() == typeof(GenericAttribute<string>)));
+            Assert.Contains(typeof(DerivesFromGenericAttribute), attributes.Select(a => a.GetType()));
+        }
+
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/56887", TestRuntimes.Mono)]
```

Comment:
Looks https://github.com/dotnet/runtime/issues/56887 is resolved, could you please remove all the `[ActiveIssue("https://github.com/dotnet/runtime/issues/56887", TestRuntimes.Mono)]`?